Source/items.cpp

```diff
@@ -688,6 +693,28 @@ void CalcPlrItemVals(int p, BOOL Loadgfx)
 						tmpac = 1;
 					bac += tmpac;
 				}
+#ifdef HELLFIRE
+				iflgs |= itm->_iFlags;
+				pDamAcFlags |= itm->_iDamAcFlags;
+				sadd += itm->_iPLStr;
+				madd += itm->_iPLMag;
+				dadd += itm->_iPLDex;
+				vadd += itm->_iPLVit;
+				fr += itm->_iPLFR;
+				lr += itm->_iPLLR;
+				mr += itm->_iPLMR;
+				ghit += itm->_iPLGetHit;
+				lrad += itm->_iPLLight;
+				spllvladd += itm->_iSplLvlAdd;
+				ihp += itm->_iPLHP;
+				imana += itm->_iPLMana;
+				dmod += itm->_iPLDamMod;
+				enac += itm->_iPLEnAc;
+				fmin += itm->_iFMinDam;
+				fmax += itm->_iFMaxDam;
+				lmin += itm->_iLMinDam;
+				lmax += itm->_iLMaxDam;
+#else

```

Comment:
If this is just to reorder a couple of values then I would prefer mindiff for hellfire